### PR TITLE
Feature/sinf 256 resize ref env

### PR DESCRIPTION
**Pull request by adrianmilne**
This is a first stab at refactoring ECS sizes - happy to refactor any/all of it.

Need to review all the instance sizes and CPU/Memory allocation - for the defaults and explicit settings - once we decide the best approach (from the options you suggested).

### CODE REVIEW

File: terraform/environments/sbx8/main.tf [hunk 1]
```diff
@@ -27,10 +27,21 @@ data "aws_ssm_parameter" "aws_account_id" {
...
+  ecr_image_id_client        = "latest"
+  client_cpu                 = 4096
+  client_memory              = 15360 #16384 - save 1GB(?) for the instance
+  client_ec2_instance_type   = "t2.xlarge" # NB: Som's initial design was 4/8, Ravi approved use of t2.xlarge as nearest instance size
+  spree_cpu                  = 4096
+  spree_memory               = 15360 #16384 - save 1GB(?) for the instance
+  spree_ec2_instance_type    = "t2.xlarge"
+  sidekiq_cpu                = 4096
+  sidekiq_memory             = 15360 #16384 - save 1GB(?) for the instance
+  sidekiq_ec2_instance_type_ = "t2.xlarge"
```
Comment (tommyb82) [inline terraform/environments/sbx8/main.tf]: Typo - `sidekiq_ec2_instance_type_`

File: terraform/modules/ecs/outputs.tf [hunk 2]
```diff
@@ -1,11 +1,3 @@
-//output "ecs_security_group_id" {
```
Comment (tommyb82) [inline terraform/modules/ecs/outputs.tf:1]: Remove commented outputs

**Review by tommyb82 — APPROVED** [review]
Excellent - certainly good enough for the moment until we confirm the sizing aspects with Som next week(?) -great work.  Only question was whether we want to try a `deployment_minimum_healthy_percent` on the service too (or at least have it built in with a sensible default, say 50% which we can then override in higher envs as required?) 🤔

File: terraform/environments/sbx8/main.tf (hunk 1 above)
Comment (adrianmilne) [reply terraform/environments/sbx8/main.tf]: oops..

File: terraform/modules/ecs/outputs.tf (hunk 2 above)
Comment (adrianmilne) [reply terraform/modules/ecs/outputs.tf:1]: Already removed in this PR 😉

**Comment by adrianmilne** [comment]
> Excellent - certainly good enough for the moment until we confirm the sizing aspects with Som next week(?) -great work. Only question was whether we want to try a `deployment_minimum_healthy_percent` on the service too (or at least have it built in with a sensible default, say 50% which we can then override in higher envs as required?) 🤔


I will add that in - you think the maximum percent should be overridable too? One value for all services, or able to configure per service?

**Comment by tommyb82** [comment]
> > Excellent - certainly good enough for the moment until we confirm the sizing aspects with Som next week(?) -great work. Only question was whether we want to try a `deployment_minimum_healthy_percent` on the service too (or at least have it built in with a sensible default, say 50% which we can then override in higher envs as required?) 🤔
> 
> I will add that in - you think the maximum percent should be overridable too? One value for all services, or able to configure per service?

If it's straightforward enough to do, then yep go for it - we'd only be able to increase it beyond 100% in clusters where we knew there was spare capacity (conversation to be had next week..) 👍

**Comment by adrianmilne** [comment]
> If it's straightforward enough to do, then yep go for it - we'd only be able to increase it beyond 100% in clusters where we knew there was spare capacity (conversation to be had next week..) 👍

Done that now - set default max to 100% and default min to 50% 👍

**Pull request merged**